From b650dfbb0b0700440cc88da31f97cec45b4140de Mon Sep 17 00:00:00 2001
From: Matthias Clasen <matthias.clasen@gmail.com>
Date: Sun, 26 Jul 2015 19:08:34 +0000
Subject: [cff] Don't use `hmtx' table for LSB (#45520).

* src/cff/cffgload.c (cff_slot_load): Use `htmx' table for advance
width only.  Bug introduced 2015-04-10.
---
diff --git a/src/cff/cffgload.c b/src/cff/cffgload.c
index 43054f8..a075ddc 100644
--- a/src/cff/cffgload.c
+++ b/src/cff/cffgload.c
@@ -3064,9 +3064,7 @@
         metrics->width  = cbox.xMax - cbox.xMin;
         metrics->height = cbox.yMax - cbox.yMin;
 
-        if ( !face->horizontal.number_Of_HMetrics )
-          metrics->horiBearingX = cbox.xMin;
-
+        metrics->horiBearingX = cbox.xMin;
         metrics->horiBearingY = cbox.yMax;
 
         if ( has_vertical_info )
--
cgit v0.9.0.2
